croncat/src/config/mod.rs

```diff
@@ -41,7 +41,20 @@ impl ChainConfig {
             .add_source(config::File::with_name(file_name))
             .build()?;
 
-        let config = settings.try_deserialize::<ChainConfig>()?;
+        let mut config = settings.try_deserialize::<ChainConfig>()?;
+
+        // Override config contract address if env var is set
+        if std::env::var("CRONCAT_CONTRACT_ADDRESS").is_ok() {
+            let contract_address = std::env::var("CRONCAT_CONTRACT_ADDRESS").unwrap();
+            config.contract_address = Some(contract_address);
+        }
+
+        if config.contract_address.is_none() {
```

Comment:
Thanks, that's awesome we can override contract addr with env var now
Small nit:
```suggestion
        if let Ok(addr) = std::env::var("CRONCAT_CONTRACT_ADDRESS") {
            config.contract_address = Some(addr);
        } else if config.contract_address.is_none() {
```

Reply:
:+1: